### PR TITLE
AB#47469 Change link to span on initial breadcrumb icon

### PR DESCRIPTION
**Pull request by phudson-he**
Change the un-required initial breadcrumb icon to a span as the "a href" link didn't do anything

### CODE REVIEW

File: arches/app/templates/views/resource/editor.htm [hunk 1]
```diff
@@ -181,6 +181,7 @@ <h1 class="page-header text-overflow ep-graph-title">
                     <div class="card-breadcrumbs">
 
                         <a href="#" class="toggle-tree"><i class="ion-arrow-left-b"></i></a>
+                        <!--<span class="toggle-tree"><i class="ion-arrow-left-b"></i></span>-->
```
Comment (aj-he) [inline arches/app/templates/views/resource/editor.htm]: Is this commented out what should be replacing the <a> ??

**Review by aj-he — COMMENTED** [review]
Please check my comment and then re-request review

File: arches/app/templates/views/resource/editor.htm (hunk 1 above)
Comment (phudson-he) [reply arches/app/templates/views/resource/editor.htm]: Yes, must have missed this, well spotted!!! I've sorted this in the latest commit

**Review by aj-he — APPROVED** [review]
looks good.

**Pull request merged**